src/msw/webview_edge.cpp

```diff
@@ -1080,6 +1099,15 @@ void wxWebViewEdge::MSWSetBrowserExecutableDir(const wxString & path)
     wxWebViewEdgeImpl::ms_browserExecutableDir = path;
 }
 
+bool wxWebViewEdge::RunScript(const wxString& javascript, wxString* output) const
+{
+    if (m_impl->m_inEventCallback)
+    {
+        wxFAIL_MSG("RunScript() cannot be used during event callbacks. Consider using RunScriptAsync()");
+    }
```

Comment:
Should we `return false` here? Or, alternatively, use
```
wxCHECK_MSG( !m_impl->m_inEventCallback, false, "..." );
```
?

Reply:
Sounds good,  obviously this shouldn't happen in release code, but still better that hanging the process.
I've implemented it with `wxCHECK_MSG()`